Api/Controllers/AttributesController.cs

```diff
@@ -17,33 +18,41 @@ public AttributesController(IAttributeGroupServices attributeServices, IGenericR
         _attributeRepo = attributeRepo;
     }
     [HttpGet]
-    public async Task<IActionResult> Get()
+    public async Task<IActionResult> GetAttributeGroups()
     {
         var attribute = await _attributeServices.List();
         return Ok(attribute);
     }
 
     [HttpGet("{id}")]
-    public async Task<IActionResult> Get(string id)
+    public async Task<IActionResult> GetAttributeGroup(string id)
```

Comment:
we use Guid as key, use Guid as type of id to take advantage of the validation feature

Reply:
```
public async Task<IActionResult> GetAttributeGroup(Guid id)
    {
        var attribute = await _attributeServices.GetById(id);
        return Ok(attribute);
    }
```

Reply:
return Ok